src/app/profile/payment-methods/payment-method/forms/add-payment-method/modal.tpl.html

```diff
@@ -0,0 +1,86 @@
+<div class="modal-content give-modal-content">
+  <div class="modal-header">
+    <div class="container-fluid">
+      <div class="row">
+
+        <div class="col-xs-12">
+          <button ng-click="$ctrl.cancel()" type="button" class="close" aria-label="Close">
+            <span aria-hidden="true">×</span>
+          </button>
+          <h4 class="modal-title border-bottom-small">Add Payment Method</h4>
+        </div>
+
+      </div>
+    </div>
+  </div>
+
+  <div class="modal-body">
+    <div class="container-fluid" tabindex="-1" role="dialog">
+      <div class="row">
+        <div class="col-xs-12">
+
+          <div class="alert alert-danger" role="alert" ng-if="$ctrl.error">
+            <p>{{$ctrl.error}}</p>
+          </div>
+
+          <div class="mb_x">
+            <div class="row">
+              <div class="col-md-6">
+                <a
+                  ng-class="{'on':$ctrl.paymentType == 'bankAccount'}"
+                  ng-click = "$ctrl.switchPaymentType('bankAccount')"
+                  class="tab-btn btn btn-lg btn-default btn-block">
+                  Bank Account
+                  <img src="/assets/img/icon-bank.png" alt="Bank Account Icon">
```

Comment:
For images, you'll need to add the imgDomain variable so they show correctly on production.

Here is an example: https://github.com/CruGlobal/give-web/blob/master/src/app/checkout/step-2/step-2.tpl.html#L12